.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js: '10'
 before_script:
-  - npx danger ci --use-github-checks
+  - DANGER_GITHUB_API_TOKEN=faketoken DANGER_JS_APP_INSTALL_ID=810101 DEBUG=* npx danger ci
```

Comment:
Do you need to run danger in every tested node version (if you want to, say, add 12 in the next week or two)? If not, you may want to use “include” to only run it once.

Reply:
Good idea, moved to its own job.